charts/cloudnative-pg/templates/rbac.yaml

```diff
@@ -155,10 +158,13 @@ rules:
   - backups
   - clusters
   - databases
+  - databases/status
   - poolers
   - publications
+  - publications/status
   - scheduledbackups
   - subscriptions
+  - subscriptions/status
```

Comment:
Are you sure this is needed here? None of the other `/status` permissions are listed here.